crates/sui-graphql-rpc/schema.graphql

```diff
@@ -2880,7 +2880,8 @@ input ObjectFilter {
 	"""
 	objectIds: [SuiAddress!]
 	"""
-	Filter for live or potentially historical objects by their ID and version.
+	Filter for live objects by their ID and version.
+	For historical versions, use multiGetObjects query instead.
```

Comment:
We do want to deprecate this input, it doesn't make a ton of sense to query live objects by their ID and version. Add a deprecation notice here (it can just be in the doc comment if the `@deprecated` support isn't there in `async-graphql`) that mentions the following:

- Why this is being deprecated (it doesn't mean what people think it means).
- What to use instead.
- When it's going away (earliest date and major version)

Check out other deprecation notices in the schema for examples:

https://github.com/MystenLabs/sui/blob/a25c0cb7e76ee00fe32c3f31ca4f381bbd55f73e/crates/sui-graphql-rpc/schema.graphql#L4344-L4359

Reply:
Added